client/coral-admin/src/routes/Moderation/containers/Moderation.js

```diff
@@ -212,7 +212,7 @@ const withModQueueQuery = withQuery(gql`
     }) {
       ...CoralAdmin_Moderation_CommentConnection
     }
-    assets: assets(query: {}) {
+    assets: assets(query: {limit: 10}) {
```

Comment:
It would be a good idea though to refactor the ModerationQueue to not need the whole asset array though.